cluster-scope/components/logging-rbac/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
```

Comment:
Instead of defining another component, I'd suggest to use the recently added [`components/monitoring-rbac`](https://github.com/operate-first/apps/tree/master/cluster-scope/components/monitoring-rbac) component.

Reply:
Monitoring and logging are diffrent Obseravbility components in OCP. BTW; To the best of my knowledge monitoring is "out-of-the-box" so I do not really understand why you need to deploy. In any event, having specific role and rolebinding will allow specific rbac control on logging, we might need that for exposure etc.

Reply:
I see what's the change. Your rolebinding uses a different `namespace` in the `subjects`. I get it now. I suggest you enhance the monitoring-rbac component rolebinding to include one additional subject for your namespace and it would grant you monitoring access to all the namespaces where we deploy the monitoring RBAC. WDYT?

Reply:
I removed the entire folder ... from what I see in 4.6 deployments this is not needed